experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -216,7 +216,23 @@ <h4>Audience</h4>
             </tr>
             <tr>
               <th>Recipe JSON</th>
-              <td colspan="3">{{ experiment.targeting }}</td>
+              <td colspan="3">
+                <div class="collapse" id="collapseRecipe">
+                  <code>
+                    <pre class="text-monospace" style="white-space: pre-wrap; word-wrap: break-word;">
+                      {{ experiment.recipe_json|linebreaks|linebreaksbr }}
+                    </pre>
+                  </code>
```

Comment:
Minor detail, but one thing I found a bit disorienting vs the current page was that there's no preview text, so it's easier to miss the recipe entirely. I don't know if it would work with bootstrap's collapse, but something to consider if it's an easy update.

Reply:
I know, in theory there's a way to do it, but I didn't want to lose a ton of time on this, and if someone needs to see the recipe they'll have to expand anyways.

Reply:
Yep, makes sense. I fiddled with it for a minute and it wasn't obvious to me, so agree it's not worth worrying about for now.